po/fr_FR.po

```diff
@@ -0,0 +1,1159 @@
+
+#: src/Util.vala:142
+msgid "Lists"
+msgstr "Listes"
+
+#: src/Util.vala:181 src/Util.vala:182
+msgid "Berry Red"
+msgstr "Rouge baie"
+
+#: src/Util.vala:184 src/Util.vala:185
+msgid "Red"
+msgstr "Rouge"
+
+#: src/Util.vala:187 src/Util.vala:188
+msgid "Orange"
+msgstr "Orange"
+
+#: src/Util.vala:190 src/Util.vala:191
+msgid "Olive Green"
+msgstr "Vert olive"
+
+#: src/Util.vala:193 src/Util.vala:194
+msgid "Yellow"
+msgstr "Jaune"
+
+#: src/Util.vala:196 src/Util.vala:197
+msgid "Lime Green"
+msgstr "Vert citron"
+
+#: src/Util.vala:199 src/Util.vala:200
+msgid "Green"
+msgstr "Vert"
+
+#: src/Util.vala:202 src/Util.vala:203
+msgid "Mint Green"
+msgstr "Vert menthe"
+
+#: src/Util.vala:205 src/Util.vala:206
+msgid "Teal"
+msgstr "Bleu sarcelle"
+
+#: src/Util.vala:208 src/Util.vala:209
+msgid "Sky Blue"
+msgstr "Bleu ciel"
+
+#: src/Util.vala:211 src/Util.vala:212
+msgid "Light Blue"
+msgstr "Bleu clair"
+
+#: src/Util.vala:214 src/Util.vala:215
+msgid "Blue"
+msgstr "Bleu"
+
+#: src/Util.vala:217 src/Util.vala:218
+msgid "Grape"
+msgstr "Raisin"
+
+#: src/Util.vala:220 src/Util.vala:221
+msgid "Violet"
+msgstr "Violet"
+
+#: src/Util.vala:223 src/Util.vala:224
+msgid "Lavander"
+msgstr "Lavande"
+
+#: src/Util.vala:226 src/Util.vala:227
+msgid "Magenta"
+msgstr "Magenta"
+
+#: src/Util.vala:229 src/Util.vala:230
+msgid "Salmon"
+msgstr "Saumon"
+
+#: src/Util.vala:232 src/Util.vala:233
+msgid "Charcoal"
+msgstr "Anthracite"
+
+#: src/Util.vala:235 src/Util.vala:236
+msgid "Grey"
+msgstr "Gris"
+
+#: src/Util.vala:238 src/Util.vala:239
+msgid "Taupe"
+msgstr "Taupe"
+
+#: src/Util.vala:416 src/Dialogs/Settings/Settings.vala:343
+msgid "Light"
+msgstr "Clair"
+
+#: src/Util.vala:419 src/Dialogs/Settings/Settings.vala:349
+msgid "Dark"
+msgstr "Sombre"
+
+#: src/Util.vala:422 src/Dialogs/Settings/Settings.vala:355
+msgid "Dark Blue"
+msgstr "Bleu sombre"
+
+#: src/Util.vala:435 src/Dialogs/Settings/Settings.vala:453
+msgid "None"
+msgstr "Aucun"
+
+#: src/Util.vala:444 src/Dialogs/Settings/Settings.vala:468
+msgid "Today + Inbox"
+msgstr "Aujourd'hui + Boîte de réception"
+
+#: src/Util.vala:616 src/Dialogs/DateTimePicker/DateTimePicker.vala:108
+msgid "Tomorrow"
+msgstr "Demain"
+
+#: src/Util.vala:618
+msgid "Yesterday"
+msgstr "Hier"
+
+#: src/Util.vala:860 src/Util.vala:929 src/Layouts/ItemRow.vala:371
+#: src/Layouts/TaskRow.vala:331 src/Views/Tasklist.vala:472
+#: src/Dialogs/Project.vala:214 src/Dialogs/Label.vala:152
+#: src/Dialogs/ReminderPicker/ReminderPicker.vala:65
+#: src/Dialogs/ProjectPicker/ProjectPicker.vala:85
+#: src/Dialogs/DateTimePicker/DateTimePicker.vala:83
+#: src/Services/MigrateV2.vala:499 src/Services/MigrateV2.vala:525
+#: src/Objects/Project.vala:456 src/Objects/Label.vala:195
+#: src/Objects/Section.vala:239
+msgid "Cancel"
+msgstr "Annuler"
+
+#: src/Util.vala:861
+msgid "Reset all"
+msgstr "Tout réinitialiser"
+
+#: src/Util.vala:900
+msgid "Welcome to Planner 3"
+msgstr "Bienvenue dans Planificateur 3"
```

Comment:
Since the tool is called Planner, isn't it better to leave the name in English? 
"Bienvenue dans Planner 3"